index.js

```diff
@@ -44,8 +44,8 @@ module.exports = class Kernel {
   //       otherwise execution succeeded
   codeHandler (code, ethInterface = new Interface(new Environment())) {
     const debugInterface = new DebugInterface(ethInterface.environment)
-
-    const instance = Wasm.instantiateModule(code, {
```

Comment:
I could but this is the incorrect API and unique to V8 for a breif time. Do you think that would be useful?